plugins/package-managers/bower/src/main/kotlin/BowerDependencyHandler.kt

```diff
@@ -68,7 +68,10 @@ private fun PackageInfo.toPackage() =
         description = pkgMeta.description.orEmpty(),
         homepageUrl = pkgMeta.homepage.orEmpty(),
         binaryArtifact = RemoteArtifact.EMPTY,
-        sourceArtifact = RemoteArtifact.EMPTY, // TODO: implement me!
+        // TODO: Get the URL for tarballs hosted on private registries. Note that the public Bower registry does not
+        //       contain any actual source (or binary) packages. Instead, it is a simple key-value store pointing from a
+        //       package name to its equivalent Git repository.
```

Comment:
Is 'equivalent' the right term here?

Reply:
I've taken this from the linked page. While I don't see anything *wrong* with the wording, I've changed it to "belonging" which probably is a better fit.